libs/wingsdk/src/target-tf-aws/counter.inflight.ts

```diff
@@ -40,6 +40,28 @@ export class CounterClient implements ICounterClient {
     return parseInt(newValue) - amount;
   }
 
+  public async dec(amount = 1): Promise<number> {
+    const command = new UpdateItemCommand({
```

Comment:
Is it possible to simply call `inc()` here with a negative value?

Reply:
Yes, I tested it and it works

I was following the [Counter Spec](https://docs.winglang.io/reference/wingsdk-spec#counter) but `inc` can be used to decrement the value

Perhaps stating that `inc` accepts both positive and negative values ​​is a better alternative

Reply:
What I meant is that we can simply implement `dec()` using `inc()`. I still think we need both methods.

```ts
async dec(amount = 1) {
  return this.inc(-1 * amount);
}
```

Reply:
I will try this but have to make some test

please do not merge